core/src/scene/filters.h

```diff
@@ -52,14 +52,15 @@ struct Filter {
         uint32_t id;
     };
     using Data = variant<none_type,
+                         Equality,
```

Comment:
@hjanetzek did you find perf benefit for these? If so can you explain me why one will see perf benefit in variant order?

Reply:
Dropped this commit